package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "relnotes",
-  "version": "0.0.0",
+  "version": "1.0.1",
```

Comment:
Shouldn't this be `1.0.0`?

Reply:
Yeah my intention was to have two commits where one bumps it to v1.0.0 and the second to v1.0.1. I now think that this is not absolutely needed so I will drop of my second commit.